packages/commerce-sdk-react/src/hooks/ShopperExperience/query.ts

```diff
@@ -14,11 +14,13 @@ import * as queryKeyHelpers from './queryKeyHelpers'
 type Client = ApiClients['shopperExperience']
 
 /**
- * Get Page Designer pages. The results will apply the visibility rules for each page's components, such as personalization or scheduled visibility.
-
-Either `categoryId` or `productId` must be given in addition to `aspectTypeId`. Because only a single page-to-product and page-to-category assignment per aspect type can be authored today, the returned results contains one element at most.
-
-**Important**: Because this resource uses the GET method, you must not pass sensitive data (payment card information, for example) and must not perform any transactional processes within the server-side scripts that are run for the page and components.
+ * Get Page Designer pages.
+ *
+ * @remarks
```

Comment:
In the docs site, I see a heading **Remarks** is the `@<param>` in this situation an enum (e.g. a limited number of valid string value options) or is it just "whatever you want to put" and then the docs site makes a heading out of it?

If it's "whatever you want" that seems a bit odd to me, and how can we make sure we're not confusing "heading titles" with "real" `@` values like `@group`?

Reply:
actually we probably don't need `@remarks`. removed